### ISSUE TITLE
How to overfit?

### ISSUE THREAD

**Issue by doglab753** [lucidrains/vit-pytorch#57]
We are trying to overfit the model on a small subset of data (1K sample), however our training and validation loss is as follows. We don't use any dropout or anything else to prevent overfitting.. Do you have any idea why this is happening?
![loss (1)](https://user-images.githubusercontent.com/76693607/103454800-82809f80-4cf8-11eb-8fef-fcb445980a4f.png)
![acc (1)](https://user-images.githubusercontent.com/76693607/103454801-84e2f980-4cf8-11eb-8130-aabb483b41f6.png)

**Comment by liberbey** [comment]
I am also trying to do the same, did you find a solution? @doglab753 @lucidrains
